lib/client.js

```diff
@@ -343,7 +359,7 @@ Client.prototype = {
       throw new Error('The get method accepts a string or array of strings.');
     }
 
-    return wrappedPostMessage.call(this, 'get', paths);
+    return wrappedPostMessage.call(this, 'get', paths).then(throwUnlessBulk.bind(null, path));
```

Comment:
Would be nice if single path `get` returned the result directly, rather than within an object. Though we'd have to make a minor release and document as a breaking change.